cmd/coder/urls.go

```diff
@@ -137,33 +219,23 @@ func (sub delSubCmd) Run(fl *pflag.FlagSet) {
 		exitUsage(fl)
 	}
 
-	if !portIsValid(port) {
+	portNum, err := validatePort(port)
+	if err != nil {
 		exitUsage(fl)
 	}
 
 	entClient := requireAuth()
-
 	env := findEnv(entClient, envName)
 
-	urlID, found := devURLID(port, urlList(envName))
+	urlID, found := devURLID(portNum, urlList(envName))
 	if found {
-		fmt.Printf("Deleting devurl for port %v\n", port)
+		flog.Info("Deleting devurl for port %v", port)
 	} else {
 		flog.Fatal("No devurl found for port %v", port)
 	}
 
-	err := entClient.DelDevURL(env.ID, urlID)
-	if err != nil {
-		flog.Error("delete devurl: %s", err.Error())
-	}
-}
-
-func (cmd urlsCmd) Spec() cli.CommandSpec {
-	return cli.CommandSpec{
-		Name:  "urls",
-		Usage: "<env name>",
-		Desc:  "get all development urls for external access",
-	}
+	err = entClient.DelDevURL(env.ID, urlID)
+	requireSuccess(err, "delete devurl: %s", err)
```

Comment:
Am I just being thick here, passing err twice? I actually want the HTTP error response to be output so it seems that I need to.

Reply:
Yes that's how I intended it to be used-- not the best looking I'll admit.